source/connect-zbt1/index.html

```diff
@@ -175,32 +181,35 @@ <h3>Form Factor</h3>
 
 <h1 class="sub-title">FAQs</h1>
 <div class="faq-list">
+  {% details "What is the difference between Home Assistant SkyConnect and Connect ZBT-1?" %} Home Assistant Connect ZBT-1 is the new name for the Home Assistant SkyConnect. Though it's a new name and look, they have the exact same hardware, capability, and support. We made this change to help make room for a new line of Connect products, specifically an upcoming Z-Wave Connect product. Going forward, whenever we mention updates or features for the Home Assistant Connect ZBT-1, this will also apply to the SkyConnect-branded devices. Retailers will continue to sell SkyConnect-branded devices while supplies last, and we would encourage you to purchase whatever is available as they are functionally identical.
+  {% enddetails %}
+
   {% details "Why include a USB extension cable?" %} USB 3.0 ports (the ones with blue on the inside) are known to cause significant noise and radio interference to any 2.4Ghz  wireless devices. This includes Zigbee and Thread. If you do not use the extension cable, it may not work at all, and if it does, it could be flaky at best with intermittent problems (issues with pairing, device dropouts, unreachable devices, timeout errors, etc).
```

Comment:
```suggestion
  {% details "Why include a USB extension cable?" %} USB 3.0 ports (the ones with blue on the inside) are known to cause significant noise and radio interference to any 2.4&nbsp;GHz wireless devices. This includes Zigbee and Thread. If you do not use the extension cable, it may not work at all, and if it does, it could be flaky at best with intermittent problems (issues with pairing, device dropouts, unreachable devices, timeout errors, etc).
```
Outside the scope of this PR, I know.